public/review.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Foo {
+  public function __construct($bar) {
+    $this->bar = $bar;
+  }
+  public function lorem() {
+    return true;
+  }
+}
+
+$foo = new Foo();
+
+if($foo->lorem()) {
+  echo 'ipsam';
+}
+$foo->hoge();
```

Comment:
__[code_sniffer]__
* Expected 1 blank line at end of file; 2 found

<!-- Comment by [SideCI](https://sideci.com) -->